### PR TITLE
Neyyi karam dosa

### PR DESCRIPTION
**Pull request by Devopslearning2025**

### CODE REVIEW

File: receipes/dosa.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 dosa flour
-karam
+extra karam
```
Comment (rameshvalapu) [inline receipes/dosa.txt:2]: do you really require extra karam

Comment (Devopslearning2025) [reply receipes/dosa.txt:2]: Yes, if extra karam extra tasty

Comment (rameshvalapu) [reply receipes/dosa.txt:2]: yeah that's nice

**Review by rameshvalapu — APPROVED** [review]
go head

**Pull request merged**